Public/Src/Deployment/tools.dsc

```diff
@@ -24,8 +24,7 @@ namespace Tools {
             ]
         };
     
-        @@public
-        export const deployed = BuildXLSdk.DeploymentHelpers.deploy({
+        const deployed = Context.getCurrentHost().os !== "win" ? undefined : BuildXLSdk.DeploymentHelpers.deploy({

```

Comment:
>Context.getCurrentHost().os !== "win" ? undefined :  [](start = 25, length = 52)

I would expect this to work on macOS. Is there a reason we aren't deploying it on that platform?

Reply:
@sahilmgandhi

Reply:
This builds the detours code, which needs the WindowsSDK dependencies and invokes cl.exe to compile, then link windows binaries. This fails on macOS.

Reply:
I've adjusted it to build for all targets, it compiles for and on macOS, so no need to force the qualifier and export the deployment as its also not packaged up. @sahilmgandhi I'm not sure what you intended to do here.

Reply:
Ahh yea i think we didn't realize/forgot to compile it for mac as well. Thanks for catching it :) 

---
In reply to: [316770978](https://github.com/microsoft/BuildXL/pull/785#discussion_r316770978) [](ancestors = 316770978)